### DISCUSSION TITLE
world switcher should always show column headers

### DISCUSSION THREAD

**Discussion by trzejos** [runelite/runelite#17676] [Ideas]
I think it would be an improvement if the World/Pop/Theme/Ping header at the top of the world switcher panel was always shown at the top. You wouldn't have to scroll all the way up to change your sort criteria. I find that I'm often in the middle of the panel somewhere if I previously had it sorted by world number or theme.

**Comment by Macweese** [comment]
I believe there exists a PR for what you describe: #16290

**Reply by trzejos** [reply]
yes this is exactly what I was hoping for
